.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
```

Comment:
delete this file

Reply:
I have removed .vscode/extensions.json  !!